src/cls/IPM/Storage/SystemRequirements.cls

```diff
@@ -41,10 +45,30 @@ Method CheckVersion() As %Status
 	}
 }
 
+/// Checks if Interoperability specification is satisfied. Health="true" implies Interoperability="enabled"
 Method CheckInteroperability() As %Status
 {
-	If (..Interoperability="enabled") && (('##class(%EnsembleMgr).IsEnsembleNamespace($Namespace)) || ('##class(%EnsembleMgr).validateNamespace($Namespace, 1))) {
-		Return $$$ERROR($$$GeneralError, "The module requires Interoperability Production Enabled. Namespace <"_$Namespace_"> doesn't support Interoperability Production")
+	If ..Interoperability = "enabled" {
+		Set tInteroperabilityRequired = 1
+	} ElseIf ..Interoperability = "disabled" {
+		Set tInteroperabilityRequired = 0
+	}  ElseIf ..Health = "true" {
+		Set tInteroperabilityRequired = 1
+	} Else {
+		Set tInteroperabilityRequired = 0
+	}
+	Set tInteroperabilityEnabled = ##class(%EnsembleMgr).IsEnsembleNamespace($NAMESPACE) && ##class(%EnsembleMgr).validateNamespace($NAMESPACE, 1)
+	If tInteroperabilityRequired && ('tInteroperabilityEnabled) {
+		Return $$$ERROR($$$GeneralError, "The module requires Interoperability Production Enabled. Namespace <"_$NAMESPACE_"> doesn't support Interoperability Production")
+	}
+	Return $$$OK
+}
+
+Method CheckHealth() As %Status
+{
+	If ..Health = "true" && '$LISTFIND($SYSTEM.Version.GetISCComponents(), "Health") {
```

Comment:
See https://github.com/intersystems/ipm/issues/364#issuecomment-2056864311 - if Health is false, we should disallow installation on IRIS for Health / HealthConnect / HealthShare.